### PR TITLE
textures: rename basis module

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

**Comment by ibgreen** [comment]
@belom88 Please merge this PR since you are focusing on textures, to make sure we don't get too many conflicts.

File: docs/whats-new.md [hunk 1]
```diff
@@ -152,25 +176,6 @@ This release adds a number of new geospatial format loaders
...
-        <p><i>TerrainLoader + <a target="_blank" href="https://deck.gl/#/documentation/deckgl-api-reference/layers/terrain-layer">TerrainLayer</a></i></p>
-      </td>
-      <td>
-        <img style="max-height:200px" src="https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/esri-i3s.gif" />
-        <p><i>I3SLoader + <a target="_blank" href="https://deck.gl/#/documentation/deckgl-api-reference/layers/tile-3d-layer">Tile3DLayer</a> + I3S</i></p>
-      </td>
-    </tr>
-  </tbody>
-</table>
-
```
Comment (kylebarron) [inline docs/whats-new.md:173]: Is this deletion intentional?

File: modules/textures/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# @loaders.gl/basis
+# @loaders.gl/textures
 
 [loaders.gl](https://loaders.gl/docs) is a collection of framework-independent 3D and geospatial parsers and encoders.
 
-This module contains loader for [basis universal textures](https://github.com/BinomialLLC/basis_universal).
+This module contains loader compressed textures and [basis universal textures](https://github.com/BinomialLLC/basis_universal).
```
Comment (kylebarron) [inline modules/textures/README.md:5]:
```suggestion
This module contains loaders for compressed textures and [basis universal textures](https://github.com/BinomialLLC/basis_universal).
```

File: modules/textures/docs/README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 # Overview
 
-The `@loaders.gl/basis` module contains a loader for Basis encoded compressed textures (images).
+The `@loaders.gl/textues` module contains a loader for Basis encoded compressed textures (images).
```
Comment (kylebarron) [inline modules/textures/docs/README.md:3]:
```suggestion
The `@loaders.gl/textures` module contains a loader for Basis encoded compressed textures (images).
```

File: modules/textures/docs/README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
 ## Installation
 
 ```bash
-npm install @loaders.gl/basis
+npm install @loaders.gl/tetures
```
Comment (kylebarron) [inline modules/textures/docs/README.md:8]:
```suggestion
npm install @loaders.gl/textures
```

**Review by kylebarron — APPROVED** [review]

File: docs/whats-new.md (hunk 1 above)
Comment (ibgreen) [reply docs/whats-new.md:173]: Hmm... Looks like I just moved it to the top to have the pretty images front and center

Comment (kylebarron) [reply docs/whats-new.md:173]: I see, but those images are specific to what was new in 2.1, no?

Comment (ibgreen) [reply docs/whats-new.md:173]: Yes but images help generate excitement about frameworks, so still worth keeping them up top :) I assume most people don't check this page on a frequent basis. 

But you have a point, I will add versions to the comments in follow-up PR.

**Pull request merged**